src/org/usfirst/frc4904/robot/commands/OverridableCommand.java

```diff
@@ -1,30 +1,29 @@
 package org.usfirst.frc4904.robot.commands;
 
 
-/**
- * This interface allows commands to easily override other commands
- */
-public interface OverridableCommand {
-	/**
-	 * sets a static override to a boolean you give it
-	 * 
-	 * @param override
-	 */
-	void setOverride(boolean setValue);
+import edu.wpi.first.wpilibj.command.Command;
+
+public abstract class OverridableCommand extends Command implements Overridable {
+	private volatile boolean isOverridden;
```

Comment:
Why is this volatile? There should not be any threading involved in overriables.